informer/disk/disk.go

```diff
@@ -12,13 +12,21 @@ import (
 	"github.com/ipfs/ipfs-cluster/api"
```

Comment:
I can't place comment before this line: The godoc description of the module needs to be updated to explain the things this informer disk informer can do.